src/SFA.DAS.AssessorService.Database/Tables/ScheduleRuns.sql

```diff
@@ -5,5 +5,6 @@
     [IsComplete] BIT NOT NULL DEFAULT 0, 
     [Interval] BIGINT NULL, 
     [IsRecurring] BIT NOT NULL DEFAULT 0, 
-    [ScheduleType] INT NOT NULL
+    [ScheduleType] INT NOT NULL, 
+    [LastRunStatus] INT NULL
```

Comment:
is the intention to populate existing schedule runs and subsequently make `LastRunStatus` NOT NULL?

Reply:
The intention is to allow null in LastRunStatus - As the newly created Schedule will have null. (Don't want to show any status  on UI if the job hasn't exectued yet).